tests/testapps/CTCatalog/CTCatalog/Tests/CTCFrameTestViewController.mm

```diff
@@ -190,26 +209,29 @@ - (void)refreshValuesForFrame:(CTFrameRef)frame {
     for (CFIndex i = 0; i < count; ++i) {
         cell = [[UITableViewCell alloc] initWithStyle:UITableViewCellStyleDefault reuseIdentifier:@"FrameLines"];
         cell.textLabel.text = [NSString stringWithFormat:@"X: %f, Y: %f", lineOrigins[i].x, lineOrigins[i].y];
-        CTFrameLineTestView* textLabel = [[CTFrameLineTestView alloc] initWithFrame:CGRectMake(0, 0, 400, 60)];
+        CTFrameLineTestView* textLabel = [[CTFrameLineTestView alloc] initWithFrame:CGRectMake(0, 0, 2 * width / 3, 60)];
         textLabel.backgroundColor = [UIColor whiteColor];
         textLabel.lineRef = static_cast<CTLineRef>(CFArrayGetValueAtIndex(lines, i));
+        CFRetain(textLabel.lineRef);
```

Comment:
this one is on purpose.  The autoreleasepool will drain before this is drawn so it needs to be retained.

---
In reply to: [79946497](https://github.com/Microsoft/WinObjC/pull/1010#discussion_r79946497) [](ancestors = 79946497)